### ISSUE TITLE
Ошибка при комплектации

### ISSUE THREAD

**Issue by Linda4ka** [Contabilizare/c5#208]
![image](https://user-images.githubusercontent.com/84187165/130311751-d0231e94-2ffb-4e94-a3e4-e6a9d9e4b354.png)
Текст ошибки:Поле объекта не обнаружено (ContractAmount)
{CommonModule.InvoiceForm.Module(177)}:		object.ContractAmount = amount;
{Document.Shipment.Form.Form.Form(357)}:	InvoiceForm.CalcTotals ( ThisObject );
Может я что-то делаю не так?

**Comment by Linda4ka** [comment]
Ошибку выдает в момент, когда пытаешься проставить галочку на товар, после, когда нажимаешь на ОК, ошибка уходит и галочка проставляется. При нажатии на следующий товар повторяется снова

**Comment by Linda4ka** [comment]
При нажатии Завершить процесс комплектации выходит:
![image](https://user-images.githubusercontent.com/84187165/130311848-34a3ff5b-5150-4161-8b87-087ea491aec2.png)
Текст ошибки:Поле объекта не обнаружено (ContractCurrency)
{Document.Shipment.ManagerModule(29)}:	obj.Fill ( Object );
{Document.Shipment.Form.Form.Form(179)}:		if ( Documents.Shipment.CreateInvoice ( CurrentObject ) ) then
{Document.Shipment.Form.Form.Form(299)}:	if ( not Write () ) then
{Document.Shipment.Form.Form.Form(291)}:		performCommand ( true );
{mngcore/MessageBox.clf(4)}:	Close(-100);

по причине:
Ошибка при выполнении обработчика - 'Filling'
по причине:
Поле объекта не обнаружено (ContractCurrency)
{CommonModule.InvoiceForm.Module(206)}:			params.ContractCurrency = object.ContractCurrency;
{CommonModule.InvoiceForm.Module(123)}:	p = getTotalParams ( Source );
{Document.Invoice.ObjectModule(57)}:	InvoiceForm.CalcTotals ( ThisObject );
{Document.Invoice.ObjectModule(47)}:		fillByShipment ();
{Document.Shipment.ManagerModule(29)}:	obj.Fill ( Object );
{Document.Shipment.Form.Form.Form(179)}:		if ( Documents.Shipment.CreateInvoice ( CurrentObject ) ) then

**Comment by Linda4ka** [comment]
При попытке просто закрыть выдает:
![image](https://user-images.githubusercontent.com/84187165/130311920-e2409061-7cae-4c21-8548-eb5790774fe7.png)
Текст ошибки:Поле объекта не обнаружено (ContractCurrency)
{Document.Shipment.ManagerModule(29)}:	obj.Fill ( Object );
{Document.Shipment.Form.Form.Form(179)}:		if ( Documents.Shipment.CreateInvoice ( CurrentObject ) ) then

по причине:
Ошибка при выполнении обработчика - 'Filling'
по причине:
Поле объекта не обнаружено (ContractCurrency)
{CommonModule.InvoiceForm.Module(206)}:			params.ContractCurrency = object.ContractCurrency;
{CommonModule.InvoiceForm.Module(123)}:	p = getTotalParams ( Source );
{Document.Invoice.ObjectModule(57)}:	InvoiceForm.CalcTotals ( ThisObject );
{Document.Invoice.ObjectModule(47)}:		fillByShipment ();
{Document.Shipment.ManagerModule(29)}:	obj.Fill ( Object );
{Document.Shipment.Form.Form.Form(179)}:		if ( Documents.Shipment.CreateInvoice ( CurrentObject ) ) then

**Comment by Linda4ka** [comment]
Короче говоря, Процесс отгрузки завершить не смогла.

**Comment by Nullarity** [comment]
> А если при начале комплектации галочки не пытаться проставлять, а просто нажать завершить, то процесс отгрузки завершается, но при этом слетели все количества и процесс был успешно завершен без кол-ва.

Да, это нормально, система создает при этом новый документ отгрузки.
